src/utils/pickItem/pickContainerApp.ts

```diff
@@ -31,5 +43,23 @@ export async function pickContainerApp(context: IActionContext, options?: PickIt
     return await runQuickPickWizard(context, {
         promptSteps,
         title: options?.title,
+        showLoadingPrompt: options?.showLoadingPrompt
+    });
+}
+
+export async function pickContainerAppWithoutPrompt(
```

Comment:
Since you're passing in the containerApp name as the context, I imagine that this will always get the container app? I feel like it should just be called `getContainerApp`.